### ISSUE TITLE
[hueemulation] Items with "TargetTemperature" tag don't show in Alexa SmartHome App

### ISSUE THREAD

**Issue by Kaskadeur111** [openhab/openhab2-addons#3139]
Upgraded to Openhab 2.2 (from 2.0)
and hueemulation binding Version 2.2

# heizung.items file:
Item **is found** in Alexa App (Smart Home):
## Number max_heizungwohnzimmer_settemp "Heizung Wohnzimmer" **["Lighting"]** {channel="max:wallthermostat:KEQ0531772:NEQ1003987:set_temp"}

Item is **not found** in Alexa App (Smart Home):
## Number max_heizungwohnzimmer_settemp "Heizung Wohnzimmer" **["TargetTemperature"]** {channel="max:wallthermostat:KEQ0531772:NEQ1003987:set_temp"}

hueemulation should expose ["Lighting"] ; ["TargetTemperature"] ; ["Switchable"] to Alexa
but afer Updating to OpenHab 2.2 without changing my item Files, all my ["TargetTemperature"] items are not found if i scan for Devices in the Alexa App.

If i change them to ["Lighting"] everything is fine except that i need to set the Temperature in % which isnt really the way it should be :-)

**Comment by ralphm2004** [comment]
It disappeared in this commit
https://github.com/openhab/openhab2-addons/commit/c205294f967639720006461fa1128cc0c75287da#diff-87266d8a7e9c5e9a1e9267f9998bde96

**Comment by wborn** [comment]
Did https://github.com/openhab/openhab2-addons/pull/2145 also fix your issue @Kaskadeur111?

**Comment by olafnorge** [comment]
@wborn How should #2145 fix the issue that temperature targets are not exposed anymore? In this PR the `TargetTemperature` tag got removed from the binding.

I can't find any reason why this happened. Was there an issue I oversaw?

@kaikreuzer, @digitaldan may you help clarifying? 

Thanks in advance!

**Comment by wborn** [comment]
Yes you're right. It seems to have been caused by #2145! Perhaps I suffered from some temporal color blindness when I made the comment so I had mistaken the deletions (red) for additions (green). I'll consult with my optician when it occurs more frequently! :nerd_face:

**Comment by olafnorge** [comment]
If I simply re-add the the removed tag the items re-appear in Alexa but if I ask her to increase/decrease the temperature Alexa tells me that it's not supported by the item.

All the items tagged as `TargetTemperature` also appear as `Lux Light` in the Alexa UI. @Kaskadeur111 I am wondering how this ever worked for you because the changes of #2145 do not seem to changed the initial behavior at all, except that items are not expose-able as `TargetTemperature` anymore.

**Comment by olafnorge** [comment]
@Kaskadeur111 if you are interested in the change just checkout https://github.com/olafnorge/openhab2-addons/tree/issue_3139 and build the addon.

**Comment by Kaskadeur111** [comment]
Vielen Dank. Ich schaus mir mal an. Und ja ich brauch das immer noch :-)Von meinem Samsung Galaxy Smartphone gesendet.
-------- Ursprüngliche Nachricht --------Von: Volker Machon <notifications@github.com> Datum: 10.11.18  16:02  (GMT+01:00) An: openhab/openhab2-addons <openhab2-addons@noreply.github.com> Cc: Kaskadeur111 <jw82@mail.de>, Mention <mention@noreply.github.com> Betreff: Re: [openhab/openhab2-addons] hueemulation ["TargetTemperature"] Items dont show in Alexa SmartHome App (#3139) @Kaskadeur111 if you are interested in the change just checkout https://github.com/olafnorge/openhab2-addons/tree/issue_3139 and build the addon.

—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/openhab/openhab2-addons","title":"openhab/openhab2-addons","subtitle":"GitHub repository","main_image_url":"https://assets-cdn.github.com/images/email/message_cards/header.png","avatar_image_url":"https://assets-cdn.github.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/openhab/openhab2-addons"}},"updates":{"snippets":[{"icon":"PERSON","message":"@olafnorge in #3139: @Kaskadeur111 if you are interested in the change just checkout https://github.com/olafnorge/openhab2-addons/tree/issue_3139 and build the addon.\r\n\r\n"}],"action":{"name":"View Issue","url":"https://github.com/openhab/openhab2-addons/issues/3139#issuecomment-437589847"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/openhab/openhab2-addons/issues/3139#issuecomment-437589847",
"url": "https://github.com/openhab/openhab2-addons/issues/3139#issuecomment-437589847",
"name": "View Issue"
},
"description": "View this Issue on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
},
{
"@type": "MessageCard",
"@context": "http://schema.org/extensions",
"hideOriginalBody": "false",
"originator": "AF6C5A86-E920-430C-9C59-A73278B5EFEB",
"title": "Re: [openhab/openhab2-addons] hueemulation [\"TargetTemperature\"] Items dont show in Alexa SmartHome App (#3139)",
"sections": [
{
"text": "",
"activityTitle": "**Volker Machon**",
"activityImage": "https://assets-cdn.github.com/images/email/message_cards/avatar.png",
"activitySubtitle": "@olafnorge",
"facts": [

]
}
],
"potentialAction": [
{
"name": "Add a comment",
"@type": "ActionCard",
"inputs": [
{
"isMultiLine": true,
"@type": "TextInput",
"id": "IssueComment",
"isRequired": false
}
],
"actions": [
{
"name": "Comment",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueComment\",\n\"repositoryFullName\": \"openhab/openhab2-addons\",\n\"issueId\": 3139,\n\"IssueComment\": \"{{IssueComment.value}}\"\n}"
}
]
},
{
"name": "Close issue",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"IssueClose\",\n\"repositoryFullName\": \"openhab/openhab2-addons\",\n\"issueId\": 3139\n}"
},
{
"targets": [
{
"os": "default",
"uri": "https://github.com/openhab/openhab2-addons/issues/3139#issuecomment-437589847"
}
],
"@type": "OpenUri",
"name": "View on GitHub"
},
{
"name": "Unsubscribe",
"@type": "HttpPOST",
"target": "https://api.github.com",
"body": "{\n\"commandName\": \"MuteNotification\",\n\"threadId\": 294723244\n}"
}
],
"themeColor": "26292E"
}
]

**Comment by davidgraeff** [comment]
OH 2.4: The tags can no freely be assigned in PaperUI or via a .cfg file

**Comment by Kaskadeur111** [comment]
Thanks. But I switched to iobroker 3 weeks ago. And guess what... I've made my complete setup I've been working on in Openhab for over 1 year. Incredible piece of software and so much more reliable and intuitive. It wasn't easy to go this way but it was 1000% the right step. Everything works out of the box flawlessly mqtt broadlink hue Alexa... Never going back. Whish you all the best and keep going. :-)
